docs/alternator/compatibility.md

```diff
@@ -16,7 +16,16 @@ provision that - Scylla requires you to provision your cluster. You need
 to reason about the number and size of your nodes - not the throughput.
 
 When creating a table, the BillingMode and ProvisionedThroughput options
-are ignored by Scylla.
+are ignored by Scylla. Tables default to a Billing mode of `PAY_PER_REQUEST`
+and `DescribeTable` API calls will return a value of 0 for the RCUs, WCUs
+and NumberOfDecreasesToday within the response.
+
+Another notable difference between DynamoDB and Scylla comes down to the
+underlying hashing algorithm. While DynamoDB uses a proprietary hashing
+function, ScyllaDB implements the well-known Cassandra Murmur3 algorithm.
```

Comment:
DynamoDB and ScyllaDB use a different underlying hashing algorithm: DynamoDB uses a proprietary hashing function, while ScyllaDB implements the well-known [Murmur3](https://docs.scylladb.com/stable/glossary.html#term-Partitioner) algorithm.

Reply:
I agree, murmur3 is not Cassandra-specific. Although technically, ScyllaDB and all our drivers implement a buggy murmur3 flavor, which is Cassandra-specific indeed :innocent: Ref: https://github.com/scylladb/scylla-rust-driver/blob/fd06928929c0dd78a72c7aca494a998da1514a79/scylla/src/transport/partitioner.rs#L27-L29

Reply:
Correct. Although I agree that removing "Cassandra" mentions should be a guidance in general. :-P

Fixed.